cvat/apps/documentation/user_guide.md

```diff
@@ -382,19 +382,23 @@ The tool consists of:
     ![](static/documentation/images/image137.jpg)
 
 ### Types of shapes (basics)
+
 There are four shapes which you can annotate your images with:
-- ``box``
-- ``polygon``
-- ``polyline``
-- ``points``
-- ``auto segmentation``
```

Comment:
Automatic segmentation will be available.

Reply:
@nmanovic 

I would not say it is a separated shape. In the new UI it was implemented just as a checkbox with tooltip.
![Screenshot from 2020-04-13 23-33-37](https://user-images.githubusercontent.com/40690378/79277228-1633e680-7eb2-11ea-84c0-d3c32718d93a.png)

Reply:
But now it’s a way to create a polygon, I thought it would be more logical to move information about automatic segmentation to the section “Annotation with polygons”?